packages/common/src/MultiVersionTestHelpers.ts

```diff
@@ -18,6 +18,21 @@ export const TESTED_CONTRACT_VERSIONS = [
   { contractType: "Perpetual", contractVersion: "2.0.1" },
 ];
 
+export const CORE_CONTRACTS_NODE_VERSION_MAPPING = {
+  "2.0.1": "0.1.0",
+};
+
+// Assumes that the contracts-node package alias is @uma/contracts-node-X-Y-Z.
+// Typical workflow is:
+// const { getAbi, getBytecode } = require(getContractsNodePackageAliasForVersion(versionString));
+export function getContractsNodePackageAliasForVerion(version: string): string {
```

Comment:
so the `version` in this function is the version of `core` which is mapped to the `contracts-node`

Reply:
Exactly. It's to help bridge the gap between modules that are already using core versions.